assets/src/components/media/manager/MediaManager.tsx

```diff
@@ -572,7 +578,9 @@ export class MediaManager extends React.PureComponent<MediaManagerProps, MediaMa
     return error.caseOf({
       just: (error) => (
         <div className="alert alert-danger fade show" role="alert">
-          {error}
+          Error: Could not upload file.
+          <br />
+          <i>{error}</i>
```

Comment:
Before, the message would just say something like "Failed to fetch", now we'll get "Error: Could not upload file." to start with every time so the user knows what happened.